### PR TITLE
Asked a question underneath

### PR DESCRIPTION
**Pull request by IChinyere**

### CODE REVIEW

**Comment by mtorpey** [comment]
Thank you for your contribution, but this isnt the sort of thing we want to include right now.

Could you rephrase your question to include the word "helicopter", please?  After that, I'll be happy.

**Comment by olexandr-konovalov** [comment]
Apologies @mtorpey - I haven't seen this PR from @IChinyere and merged #11 which incorporates the same commit (also squashed). I suggest to closes this OR now and create an issue to address the helicopter suggestion.
